srcpkgs/Quaternion/template

```diff
@@ -1,9 +1,8 @@
 # Template file for 'Quaternion'
 pkgname=Quaternion
-version=0.0.95.beta5
+reverts="0.0.95.beta5_1"
```

Comment:
No need for a revert, `q-0.0.95.beta5_1` < `q-0.0.95_1` (you can use e.g. `xbps-uhelper cmpver` to compare two versions, 255 (-1) as result means first version is smaller, 1 first version is bigger and 0 if versions are the same

Reply:
TIL, thanks!